packages/codegen/package.json

```diff
@@ -10,7 +10,7 @@
     "knex": "^0.21.4",
     "is-plain-object": "^3.0.1",
     "pg": "^8.3.0",
-    "pg-structure": "^5.10.13",
+    "@homebound/pg-structure": "^7.2.4",
```

Comment:
...I forgot, does npm let us do `7.2.3-1` How to properly semvers forks is nuanced. I'm good with this though.